server/repos.go

```diff
@@ -175,7 +175,12 @@ func PostRepo(c *gin.Context) {
 		httputil.GetURL(c.Request),
 	)
 
-	// TODO(bradrydzewski) verify repo not exists
+	_, err_ := store.Repo(name)
+	if err_ != nil {
+		//c.Fail(409, err_)
+		c.Fail(409, fmt.Errorf("Conflict"))
```

Comment:
in this case we can use `c.String(409, "Repository already exists")` ... just shortens the code a bit